modules/ROOT/pages/configure-client-management-azure.adoc

```diff
@@ -0,0 +1,70 @@
+= Configure Azure Client Management
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can use dynamic client registration to configure Azure client management with Anypoint Platform.
+
+include::partial$include-nav-steps-org-admin.adoc[]
+
+include::partial$include-nav-client-providers.adoc[]
+
+. Click *Add Client Provider*, and then select *OpenID Connect DCR for Azure*.
++
+The *Add OIDC Azure client provider* page appears.
++
+. Enter a name and description for your client provider.
+. Enter the following values from your identity provider’s configuration:
+.. *Issuer*: URL that the OpenID provider asserts is its trusted issuer.
+.. *Tenant ID for Azure AD*: The tenant ID from Azure Active Directory. For more information on obtaining your tenant ID, see https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-how-to-find-tenant[How to find your Azure Active Directory tenant ID].
+.. *Authorization Header*: The header to provide credentials to authenticate with a server. This header is required if the provider restricts registration requests to authorized clients.
```

Comment:
We can add the auth-header format here, For Azure AD, this value is Bearer ${api_token}, where api_token is an API token created through Azure.

Reply:
```suggestion
.. *Authorization Header*: The header provides credentials to authenticate with a server. This value is the bearer `${api_token}`, where `api_token` is an API token created through Azure. This header is required if the provider restricts registration requests to authorized clients. 
```